oneandone/src/main/java/org/apache/jclouds/oneandone/rest/compute/function/HardwareFlavourToHardware.java

```diff
@@ -53,7 +53,7 @@ public Hardware apply(HardwareFlavour from) {
       }
       final HardwareBuilder builder;
       builder = new HardwareBuilder()
-              .ids(from.id())
+              .ids(from.name())
```

Comment:
maybe better to use 
```
.id(from.id())
.name(from.name())
```
if `.ids()` is not what you want here

Reply:
My first approach was what you have suggested, for some reason the API has similiar Id's between the baremetal flavors and the cloud flavors, the only thing to make them unique is the name so far, this might be an issue on the API side that might get changed in the future, if that ever happens we can go with what you suggested